framework/tests/viewpointtest/VarargType.java

```diff
@@ -0,0 +1,27 @@
+import viewpointtest.quals.*;
```

Comment:
Can you add the usual starter text with a link to the issue that this test is for?

Reply:
Tests that actually use some annotations would be nice.

Reply:
>Can you add the usual starter text with a link to the issue that this test is for?

Done

> Tests that actually use some annotations would be nice.

Done